foolbox/plot.py

```diff
@@ -68,5 +70,10 @@ def images(
             ax.set_yticks([])
             ax.axis("off")
             i = row * ncols + col
+            if labels is not None:
+                ax.set_title(labels[i])
             if i < len(x):
                 ax.imshow(x[i])
+                
+    if return_fig:
+        return fig, axes
```

Comment:
This lets the user get the figure back and mess with it themselves if they want. Default disabled as not to spam Jupyter notebooks with the output if this is not desired.